pkg/kmm/containers_test.go

```diff
@@ -0,0 +1,230 @@
+package kmm
+
+import (
+	"testing"
+
+	v1 "github.com/NVIDIA/gpu-operator/api/v1"
+	"github.com/rh-ecosystem-edge/kernel-module-management/api/v1beta1"
+	"github.com/stretchr/testify/assert"
+)
+
+func buildRegExKernelMapping(regexp string) *v1beta1.KernelMapping {
```

Comment:
Could you please move all the help functions to the bottom

Reply:
Moved! Thanks